components/dropdownlist/filter.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Filter
+page_title: DropDownList - Filter
+description: Filtering in the DropDownList for Blazor.
+slug: components/dropdownlist/filter
+tags: telerik,blazor,drop,down,list,dropdownlist,filter
+published: True
+position: 5
+---
+
+# DropDownList Filter
+
+The Filter in the DropDownList component is an input shown at the top of the dropdown popup, used for filtering the underlying datasource.
```

Comment:
Try adding some business value to this sentence, use the combobox article as base.

Reply:
Done.